### ISSUE TITLE
Kache ,Gson , Cant Convert  it Back To Kache Object  ,  build Error

### ISSUE THREAD

**Issue by blackholeearth** [MayakaApps/Kache#289]
after i restart the i app i want to access same memory cache.    
 
 is it possible??

**Comment by sschuberth** [comment]
> after i restart the i app i want to access same memory cache.
> 
> is it possible??

Not really, I believe. As it's a "in-memory" cache, it persists only while the application runs. On restart, the in-memory cache will be empty again. If you want to persist data across restarts, use a `FileKache`.

**Comment by blackholeearth** [comment]
thank i some how fixed the gson error .i used something else..

using  memory kache it works well...
---

i was stuck at  file persistant cache...

 iam reading your FileKache answer  here; https://github.com/MayakaApps/Kache/issues/288#issuecomment-2797253359
